pom.xml

```diff
@@ -142,6 +142,16 @@
 					</annotationProcessorPaths>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>com.google.cloud.tools</groupId>
```

Comment:
If this plugin is added to create an image then you dont need it.

Reply:
deleted